sisyphus/worker.py

```diff
@@ -153,6 +154,8 @@ def stop(self):
 def worker(args):
     # Change job into error state in case of any exception
     gs.active_engine = gs.engine().get_used_engine(args.engine)
+    sisyphus.toolkit._sis_running_in_worker = True
```

Comment:
Technically I guess we should also set this back to false in a finally block afterwards.

Reply:
I don't think it's relevant at the moment, but you are right. I added it.